tests/firedrake/macro/test_macro_solve.py

```diff
@@ -151,7 +151,7 @@ def test_stokes(mh, variant, mixed_element):
             Z,
             [Z.sub(0), VectorSpaceBasis(constant=True, comm=COMM_WORLD)]
         )
-        solve(a == L, zh, bcs=bcs, nullspace=nullspace, solver_parameters={"ksp_type": "preonly", "pc_type": "cholesky", "pc_factor_mat_solver_type": "mumps"})
+        solve(a == L, zh, bcs=bcs, nullspace=nullspace)
```

Comment:
There were some redudant flops caused by poor handling of gem.Deltas into dense identity matrices in code generated for `VectorElement`. Apparently MUMPS is happy after restoring the original behavior without the dense identity https://github.com/firedrakeproject/fiat/pull/166

Reply:
Ah cool. I'm happy then.